src/main/java/duke/Main.java

```diff
@@ -0,0 +1,39 @@
+package duke;
+
+import java.io.IOException;
+
+import duke.ui.MainWindow;
+import javafx.application.Application;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Scene;
+import javafx.scene.layout.AnchorPane;
+import javafx.stage.Stage;
+
+/**
+ * A GUI for Duke using FXML.
+ */
+public class Main extends Application {
+
+    private final Duke duke = new Duke();
```

Comment:
While non-essential for private class members, perhaps you can still consider adding JavaDoc comments for them as it may be beneficial for you when a need to debug your code arises. 

These comments can be specified on a single line as recommended in our textbook: 
https://se-education.org/guides/conventions/java/intermediate.html#comments 
under the comments segment!